binding/Binding/SKCanvas.cs

```diff
@@ -240,7 +240,7 @@ public void DrawText (string text, float x, float y, SKPaint paint)
 			if (paint == null)
 				throw new ArgumentNullException ("paint");
 
-			var bytes = System.Text.Encoding.UTF8.GetBytes (text);
+			var bytes = Util.GetEncodedText(text, paint.TextEncoding);
```

Comment:
Well, I personally recommend to use [CoreFX coding style](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md) since it's supported by default by tools like visual studio and R# and more common in general, but it's up to you.